src/purerpc/protoc_plugin/plugin.py

```diff
@@ -10,7 +10,11 @@
 def generate_import_statement(proto_name):
     module_path = proto_name[:-len(".proto")].replace("-", "_").replace("/", ".") + "_pb2"
     alias = get_python_module_alias(proto_name)
-    return "import " + module_path + " as " + alias
+    if "." in module_path:
+        parent_modules, sub_module = module_path.rsplit(".", 1)
+        return "from " + parent_modules + " import " + sub_module + " as " + alias
```

Comment:
please add a comment about trying to match the import style of existing generators, for the sake of some post-processing tools

Reply:
I'll gladly add a comment in the code. I am not sure whether it will help any other user (other than me) immediately but wrong import statements are common problem with gRPC.